mecra.csl

```diff
@@ -0,0 +1,1467 @@
+    <updated>2019-08-07T15:51:11+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="collection-editor">ed.</term>
+      <term name="editorial-director">ed.</term>
+    </terms>
+  </locale>
+  <locale xml:lang="tr">
+    <terms>
+      <term name="page-range-delimiter">-</term>
+      <term name="editor" form="verb-short">ed.</term>
+      <term name="translator" form="verb-short">çev.</term>
+      <term name="translator" form="short">çev.</term>
+      <term name="editortranslator" form="verb-short">ed. &amp; çev.</term>
+      <term name="editortranslator" form="verb">ed. &amp; çev.</term>
+      <term name="in"/>
+      <term name="volume" form="short">c.</term>
+      <term name="volume">cilt</term>
+      <term name="et-al">v.dğr.</term>
+      <term name="collection-editor">thk.</term>
+      <term name="no date" form="short">t.y.</term>
+      <term name="editor" form="verb">ed.</term>
+      <term name="translator" form="verb">çev.</term>
+      <term name="director" form="verb">haz.</term>
+      <term name="director" form="short">haz.</term>
+      <term name="director" form="verb-short">haz.</term>
+      <term name="container-author" form="verb-short"/>
+      <term name="edition" form="short">bs.</term>
+      <term name="editorial-director">nşr.</term>
+      <term name="editorial-director" form="verb-short">nşr.</term>
+      <term name="editorial-director" form="short">nşr.</term>
+    </terms>
+  </locale>
+  <macro name="editor-translator">
+    <group delimiter=", ">
+      <group delimiter=" ">
+        <choose>
+          <if variable="container-author reviewed-author" match="any">
+            <group>
+              <names variable="container-author reviewed-author">
+                <label form="verb-short" text-case="lowercase" suffix=" "/>
+                <name and="symbol"/>
+              </names>
+            </group>
+          </if>
+        </choose>
+      </group>
+      <names variable="editor collection-editor editorial-director director" delimiter=", ">
+        <label form="verb-short" text-case="lowercase" suffix=" "/>
+        <name and="symbol"/>
+      </names>
+      <choose>
+        <if type="chapter" match="none">
+          <names variable="translator">
+            <label form="verb-short" suffix=" "/>
+            <name/>
+          </names>
+        </if>
+      </choose>
+    </group>
+  </macro>
+  <macro name="secondary-contributors-note">
+    <choose>
+      <if type="paper-conference entry-dictionary entry-encyclopedia chapter" match="none">
+        <text macro="editor-translator"/>
+      </if>
+      <else-if type="chapter" match="any">
+        <text term="translator" form="verb-short" suffix=" "/>
```

Comment:
You'll want to handle this with a label just like in line 64-66. This produces incorrect output (check the sample citation for Mares in the auto-generated example on github).